### ISSUE TITLE
django save issue in testing channel

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#1551]
Thanks to forum member rockmar for reporting this 3.8.15-11 issue. It would seem that in addition to the already altered save order to cater for our new django v 1.8.16 strickter behaviour:
https://github.com/rockstor/rockstor-core/pull/1546/commits/fd59e32b22ec0d3e44c35b7c52eb0703c29482b4
there appears to be a related issue in pool.py:
```
File "/opt/rockstor/src/rockstor/rest_framework_custom/generic_view.py", line 40, in handleexception
yield
File "/opt/rockstor/src/rockstor/storageadmin/views/pool.py", line 268, in post
p.disk_set.add(*disks)
File "/opt/rockstor/eggs/Django-1.8.16-py2.7.egg/django/db/models/fields/related.py", line 750, in add
obj.save()
File "/opt/rockstor/eggs/Django-1.8.16-py2.7.egg/django/db/models/base.py", line 685, in save
"unsaved related object '%s'." % field.name
ValueError: save() prohibited to prevent data loss due to unsaved related object 'pool'.
```
Please update the following forum thread with this issues resolution:
https://forum.rockstor.com/t/unable-to-create-pools-3-8-15-11-unsaved-related-object/2420

**Comment by phillxnet** [comment]
Having a quick look at this one now.

**Comment by phillxnet** [comment]
With the above added p.save() command/commit a pool was successfully created in an identical scenario that had previously reproduced the original reporters error.

**Comment by phillxnet** [comment]
Closing as fixed by #1552 :
https://github.com/rockstor/rockstor-core/commit/f01345b599e9a33495efcd6d63c42ebf3bd4712e
and:
https://github.com/rockstor/rockstor-core/commit/035ed73bb120aee5320898d6395e6bbdcddb6361
